packages/app/src/app/pages/Sandbox/Editor/Content/Tabs/ModuleTab/ModuleTab.tsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import EntryIcons from 'app/pages/Sandbox/Editor/Workspace/Files/DirectoryEntry/Entry/EntryIcons';
+// eslint-disable-next-line import/extensions
+import getType from 'app/utils/get-type.ts';
```

Comment:
@Saeris You can just delete the `eslint-disable` line and delete the `.ts` extension here